TLM/TLM/Patcher.cs

```diff
@@ -38,9 +38,9 @@ public class Patcher {
                 Log.Info($"Harmony attribute-driven patching successfull!");
             }
             catch (Exception e) {
-                Log.Error("Could not deploy Harmony patches");
-                Log.Info(e.Message);
-                Log.Info(e.StackTrace);
+                Log.Error("Could not apply Harmony patches because the following exception occured:\n " +
+                    e +
+                    "\n   -- End of inner exception stack trace -- ");
```

Comment:
The stack trace from error would be appended to the end of the log

Reply:
Yeah, exception.ToString() is much better than the alternatives, since we have all the infos right there. No need to do basically the same with e.Message.ToString() + e.StackTrace.ToString().

Reply:
One problem: have you tried causing an error?
Re-thrown exception from CitiesHarmony doesn't put any useful information where was the problem. It's general info: "...an error occurred while patching, blah, blah, blah..."
I noticed it when I've been experimenting with migration of TrainAI redirection to harmony and I renamed parameter names. Back then I've had to run debugger to get any info what was wrong